__tests__/outdated-snapshot-reporter.spec.js

```diff
@@ -35,7 +36,7 @@ describe('OutdatedSnapshotReporter', () => {
 
     const commonTest = `
     const fs = require('fs');
-    const {toMatchImageSnapshot} = require('${jestImageSnapshotDir}');
+    const {toMatchImageSnapshot} = require('${jestImageSnapshotDir.replace(/\\/g, '/')}');
```

Comment:
hmm i thought path.join used platform specific separators (`/`). Is that not what youre seeing?

Reply:
It does, but we always want forward slash here, even on windows. This is because `require` uses forward slashes on all platforms